### ISSUE TITLE
request to become a maintainer

### ISSUE THREAD

**Issue by taylorcode** [protobufjs/protobuf.js#1581]
My team maintains a fork of protobufjs here https://github.com/dropbox/protobuf.js because we have not been able to merge in fixes and improvements through the PR process.

https://github.com/protobufjs/protobuf.js/issues/1228
https://github.com/protobufjs/protobuf.js/issues/1249
https://github.com/protobufjs/protobuf.js/pull/1234
https://github.com/protobufjs/protobuf.js/pull/1463
https://github.com/protobufjs/protobuf.js/pull/1460
https://github.com/protobufjs/protobuf.js/pull/1461

Without these changes, especially #1461, we cannot use protobufjs.

As far as I know there is no alternative to protobufjs that really works for our use case (in the browser). google-protobuf is 230kb (min) 47kb (min + gzip) according to [bundlephobia](https://bundlephobia.com/result?p=google-protobuf@3.16.0-rc.1) and generates highly bloated code, while protobufjs-minimal is 22k (min) and 6k (min + gzip) and generates very lean code. Going forward we plan to fully embrace proto and make it the standard format for all data, api, configuration, etc.

I'd like to become a maintainer on this project primarily to help incorporate the changes we need and help ensure that this project stays alive.

**Comment by alexander-fenster** [comment]
Hi @taylorcode,

[I'm speaking for myself here, not for my employer.]

First of all, I acknowledge that it takes a lot of time for the things to go through - we were in the same situation two years ago :) so @dcodeIO granted access to us and we managed to set up auto releases, merge some critical changes, and we do spend as much time as we could to keep the light on (though, as you understand, it's not a primary job for neither of us).  I personally would love to have as much help as possible, so as long as @dcodeIO won't object, I see no problems with making you a maintainer here. After all, it's open source and we're not the owners here.

Now back to the list of PRs, I'm working on a proper proto3 JSON support to be included in v7.0 (expect one or two more weeks) and then we'll be able to release v7.0.0 (that will include #1234 and finally split the CLI and the main package). Some of the other PRs are in blocked state because we as reviewers didn't have access to editing your branch (so we can't hit the Update branch button and verify that tests pass).

Long story short - we're in the same boat here, and I'll be happy to collaborate.

**Comment by danielhochman** [comment]
+1 to adding more maintainers, and @taylorcode seems like a great candidate.

We're using pbjs for [lyft/clutch](https://github.com/lyft/clutch). It integrates perfectly with our development flow and standardization on protos. However, we have run into a number of significant issues, many with existing stale PRs. We have been able to hack or otherwise work around many problems solely to avoid having to rely on a fork. Personally, I have shied away from contributing since my confidence that it would result in a merged fix is low. Adding more maintainers and cleaning up stale issues and PRs would really improve the overall health and outlook of the project, and should ultimately lessen the burden on each individual maintainer.

**Comment by alexander-fenster** [comment]
I granted write access to @taylorcode and hope you will join us in triaging issues :)

**Comment by taylorcode** [comment]
I'm thrilled. Thank you!
